.circleci/config.yml

```diff
@@ -59,6 +59,40 @@ jobs:
             - pace-util/.tox
             - pace-util/venv
 
+  test_dsl:
+    docker:
+      - image: cimg/python:3.8
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+          name: Install Submodules
+          command: git submodule update --init
+      - restore_cache:
+          keys:
+            - v2-{{ checksum "constraints.txt" }}-{{ checksum "dsl/setup.py" }}
+      - run:
+          name: run tests
+          command: |
+            if [ $(python changed_from_main.py dsl) != false ]; then
+              pip3 install --upgrade setuptools wheel
+              python3 -m venv dsl/venv
+              . dsl/venv/bin/activate
+              pip3 install -r dsl/requirements.txt -c constraints.txt
```

Comment:
Where is `dsl` installed? I don't see a command for it.

Reply:
`dsl` is just the dsl directory in pace.

Reply:
Added it!